src/client/eppo-client.spec.ts

```diff
@@ -208,6 +214,158 @@ describe('EppoClient E2E test', () => {
       jest.restoreAllMocks();
     });
 
+    describe('get*AssignmentDetails', () => {
+      it('should set the details for a matched rule', () => {
+        const client = new EppoClient(storage);
+        client.setIsGracefulFailureMode(false);
```

Comment:
This could probably be in a `beforeAll()` and then reset storage in a `beforeEach()`